### DISCUSSION TITLE
Will hosts file without IP addresses be problem?

### DISCUSSION THREAD

**Discussion by nonetrix** [0xERR0R/blocky#655] [Q&A]
I found quite a good hosts file for blocking phising domains https://phishing.army/ and https://phishing.army/download/phishing_army_blocklist.txt however it doesn't have something like
```
0.0.0.0 domain
127.0.0.1 domain
```
instead just 
```
domain
domain
```
This doesn't even work in /etc/hosts will it work with blocky?

**Comment by 0xERR0R** [comment]
Yes, you can use this list as a blacklist in blocky.  Blocky supports plain domains list, host format and regex, see https://0xerr0r.github.io/blocky/configuration/#blocking-and-whitelisting
